assets/src/edit-story/app/index.js

```diff
@@ -69,6 +70,7 @@ function App({ config }) {
                               <DropTargetsProvider>
                                 <GlobalStyle />
                                 <DevTools />
+                                <StatusCheck />
```

Comment:
Is there a reason for the `StatusCheck` to be so deep in the layout? We could probably move it upper.